typographie/forms.py

```diff
@@ -0,0 +1,21 @@
+from django import forms
+from tinymce.widgets import TinyMCE
+from typographie.admin import typographie
+
+
+class TypographieFilterForm(forms.Form):
+    class Media:
+        js = ("/static/tinymce/tinymce.min.js",)
+
+    text_to_filter = forms.CharField(widget=TinyMCE(attrs={"cols": 80, "rows": 30}))
+
+    def __init__(self, *args, **kwargs):
+        super(TypographieFilterForm, self).__init__(*args, **kwargs)
```

Comment:
C'est l'ancienne méthode pour appeler `super`. Désormais il faut le faire comme ceci : `super().__init__(*args, **kwargs)`

Reply:
Modifié et noté éviter de refaire l'erreur 👌
bf439554fa7087766b12dac22d07334fd59af0ae

Reply:
ok c'est pas tant une erreur en soit car la syntaxe est toujours supportée mais sera dépréciée à l'avenir